csstest.js

```diff
@@ -85,7 +85,19 @@ var Test = function (spec) {
 					href: devLinkFormat(spec.tests.links),
 					target: '_blank',
 					textContent: 'DEV',
-					className: 'spec-link'
+					className: 'spec-link w3c-link'
```

Comment:
One spec, the fullscreen API, is defined by the WHATWG, not the W3C.
I'm not sure if you should add a dedicated icon for it, since it's the only spec in this case.

Reply:
Oh, this true.

Reply:
It's now possible to add more logos for DEV